correct grammar, move prod usage (#20)
mathieutu/vue-cli-plugin-express: README.md

**:star: Features:**

- Fully customizable Express Server:
  - Just add your api routes into your project (with import/export support)
  - Optional automatic fallback to the Vue app. Serve both the api and the application with Express
  - Optional socket.io support

> - :soon: Example routes and components

Reviewer: ```suggestion
- 🔜 Example routes and components ([⚠️ Needs contribution!](https://github.com/mathieutu/vue-cli-plugin-express/issues/12))
```

:warning: Make sure you have vue-cli 3.x.x:

```sh
vue --version
```

If you have not installed vue-cli 3, first follow the install instructions here: [https://cli.vuejs.org/](https://cli.vuejs.org/)

Generate a project using vue-cli 3

```sh
vue create my-new-app
```

Navigate to the newly created project folder and add this plugin. It is good practice to commit your changes prior to installing a new plugin.

```sh
cd my-new-app
vue add express
```

> :soon: _:information_source: An example `APIExample.vue` component will be added should you choose to include examples when installing the plugin._

Reviewer: ```suggestion
:soon: _An example `APIExample.vue` component will be added should you choose to include examples when installing the plugin. ([⚠️ Needs contribution!](https://github.com/mathieutu/vue-cli-plugin-express/issues/12))_
```

## Usage

To start your server for development, run:

```sh
yarn express
```

The server will watch for file changes and automatically restart.

Then you just have to start the web app:

```sh
yarn serve
```

Alternatively you can serve the web app and api together from express by setting the plugin option, `shouldServeApp` to `true`. See the [Configuration](#configuration) section below for further details. You will also need to tell Express where to serve your web application by adding the following to `./srv/index.js`;

```js
var router = express.Router();
router.use(express.static("./dist"));

> app.use("/", router);

Reviewer: All the idea of this package is to avoid that (also, this will not work with vue-router)! `shouldServeApp` to true (chich is the default) is sufficient to serve properly the app.

Could you remove this part please?

Author: I added this as I found it necessary to serve my front-end code. I have also got vue-router working with it.
In my project, `shouldServeApp` is set to true but when I run `yarn express`  it tells me no routes available and the server isn't serving anything on **localhost:3000\**
I've tried this a few times with a fresh project and only the express plugin added but could not make it work.
I've now looked closer at your `server.js` file and realise the vue app is only served when in production mode. I'll update the documentation to show that as it tripped me up.

On another subject, I was going to create a PR to allow setting the Express port from an environment variable `$PORT`. Or is there an existing way to do this?

Author: I have removed the code example

}
```

> ### Production

Reviewer: ```suggestion
## Use your app in production
```

Reviewer: Here I prefer the previous formula, at the first level.

For production use you need to move `@vue/cli-service` from `devDependencies`to `dependencies`.

> Then build your app and run the server;

Reviewer: ```suggestion
Then build your app and run the server:
```
